docs/whatsnew.rst

```diff
@@ -95,80 +95,47 @@ Technical novelties in eXo Platform 5.1
 
 **Upgrades**
 
-In the version 5.0, many tools/applications used by eXo Platform were 
-upgraded to newer  versions in order to benefit from their latest 
-updates and improvements: 
+In 5.1 version, eXo Platform upgraded to Jboss EAP 7.1 in order to:
 
-- Tomcat 8.5
+-  Benefit from its lastest improvements. Learn more about them from this 
+   `link <https://www.redhat.com/en/blog/red-hat-releases-jboss-eap-71>`__.
 
-- JBoss EAP 7.0
+-  Fix some issues occuring with the previous version:
 
-- Elasticseach 5.6 
-
-- Infinispan 8
-
-- Groovy 2.4
-
-- JQuery 3.2 
-
-- Shindig 2.5
+  -  "Remember me" token not recognized in Jboss EAP 7.0
+  
+  -  Instance id of a cluster node is not added on the JSESSIONID cookie
 
 **Performances and scalability**
 
-In eXo Platform 5.0, we insisted on improving the performance and the 
-scalability by changing some strategies, configurations and data 
-storage.  
-
-- Cache strategy and configuration
-
-- New cache on organization service
-
-- Files indexing in Elasticsearch
-
-- Storage of all spaces members in Social database tables
+In eXo Platform 5.1 we continue our effort on improving the performance and the scalability
+by changing some strategies, configurations and data storage:
 
+-  Users/Groups/Roles import in database: a new job is implmentd to import IDM data 
+   in IDM database asynchronously and periodically.
+  
+-  Smarter data user synchronization: only mandatory data are imported in database during synchronization, 
+   other user's data are imported on login or when it is required (lazy import).
+  
 **Migrations**
 
-With eXo Platform 5.0, some data were migrated from JCR to database to 
-improve performance and scalability:
-
-- Notifications and settings in JPA dataset
-
-- ECMS files are indexed in Elasticsearch
-
-- Space members stored in Social database tables 
-
-Many other improvements and refinements were also done in :
-
-- Customizing skin: No more need to duplicate the whole project 
-  platform-ui.
-
-- Extending the IM list of user profile is now doable through UI.
-
-- When indexing JCR workspace, old indexes are always existing until the 
-  end of indexation.
-
-- An Mbean is added to reload Javascript.
-
-- A new minify attribute allowing to enable/disable the minification.
+-  As cited in the previous section, eXo Chat has a new design. 
+   This new design is powered by the frontend framework Vue.js instead of WebUI, JQuery and Juzu 
+   which are now considered as obselete and not enough productive.
```

Comment:
I don't think we should write this. It is very negative and most of the platform still uses these technologies. I am not sure even sure it makes sense to talk about vue.js.
But if you want to talk about it, I would just say "As cited in the previous section, eXo Chat has a new design. Moreover, from a technical point of view, it has been implemented using a new frontend framework (Vue.js)."
WDYT ?

Reply:
OK